lib/HeroBonus.h

```diff
@@ -253,13 +253,13 @@ struct DLL_LINKAGE Bonus
 {
 	enum { EVERY_TYPE = -1 };
 
-	enum BonusType
+	enum BonusType : ui8 //TODO: at 2015:10:24 there are 149 possible values, how about EVERY_TYPE = -1?

```

Comment:
I have plans to make this 32 bit (as all other IDs), we may limit to 16 bit, but 8 bit is not enough for mod support